lib/matplotlib/cbook/__init__.py

```diff
@@ -2030,24 +2030,25 @@ def _pformat_subprocess(command):
 
 def _check_and_log_subprocess(command, logger, **kwargs):
     """
-    Run *command* using `subprocess.check_output`.  If it succeeds, return the
-    output (stdout and stderr); if not, raise an exception whose text includes
-    the failed command and captured output.  Both the command and the output
-    are logged at DEBUG level on *logger*.
+    Run *command*.  If it succeeds, return the its stdout output; if not, raise
```

Comment:
s/the its/its/

but also docstrings should be split after the first sentence.

Reply:
thanks; reworded the docstring.